### PR TITLE
Updated devtools_shared dep missing bumped to 0.9.3+1

### PR DESCRIPTION
**Pull request by terrylucas**

### CODE REVIEW

**Comment by terrylucas** [comment]
Would like to update all pubspec.yaml files to

`version: 0.9.3+1
`

File: packages/devtools_server/pubspec.lock [hunk 1]
```diff
@@ -243,4 +243,4 @@ packages:
     source: hosted
     version: "0.7.3"
 sdks:
-  dart: ">=2.10.0-110 <=2.11.0-177.0.dev"
+  dart: ">=2.10.0-110 <=2.11.0-161.0.dev"
```
Comment (devoncarew) [inline packages/devtools_server/pubspec.lock:246]: Not for this PR, but I'm not sure why we so tightly control which SDKs the server can install into.

**Review by devoncarew — APPROVED** [review]

File: packages/devtools_server/pubspec.lock (hunk 1 above)
Comment (terrylucas) [reply packages/devtools_server/pubspec.lock:246]: Me either I was surprised to see this.

Comment (devoncarew) [reply packages/devtools_server/pubspec.lock:246]: Ah! This is the lock file - I thought it was the pubspec. The pubspec sdk constraint is what will matter to users - that one looks good.

**Review by kenzieschmoll — COMMENTED** [review]
did you use the update_version.sh script to do this? I don't see the change to the devtools.dart file

**Comment by terrylucas** [comment]
> did you use the update_version.sh script to do this? I don't see the change to the devtools.dart file

Not yet wanted to make sure that the +1 was how we wanted to rev.

**Comment by kenzieschmoll** [comment]
+1 sgtm

**Comment by terrylucas** [comment]
Everything is updated.

**Review by kenzieschmoll — APPROVED** [review]

**Pull request merged**